UPDATING.md

```diff
@@ -34,14 +34,18 @@ Make sure your `cldr-json` directory is otherwise clean (`git status`)
 
 ### Publishing
 
-1. Run the script `cldr-generate-zip.sh` to generate zipfiles under `dist/` - these are suitable for uploading to a release page
-2. create a release tag such as `43.0.0` in this repository. or `43.0.0-ALPHA2`.  Create a GitHub release, use other [releases](https://github.com/unicode-org/cldr-json/releases) as a guide.
-3. npm packages can be updated as well. Each sub-subdirectory of `cldr-json/cldr-json` is a separate npm package. The following script will preview
+1. Run the script `cldr-generate-zip.sh` to generate zipfiles under `dist/`, then rename them by inserting the release tag between the two hyphens; for example, rename `cldr--json-full.zip` to `cldr-44.0.0-ALPHA2-json-full.zip`.
```

Comment:
Not correct.

`VERSION` should be set in your local config .sh file to the correct version such as `44.0.0-ALPHA2` and then the zip file will be created correctly.

IN fact it looks like the npms published were published as `44.0.0` which is final data. This isn't correct and we'll have to deprecate them or unpublished them and have the final version be 44.0.1 or some such.

Reply:
"published as 44.0.0 which is final data" -- I did include the "-ALPHA2" in the names; sorry I'm not quite getting it -- sounds like UPDATING.md lacks instructions about setting VERSION? Or did I miss that?